### PR TITLE
Etcd clustering

### PR DESCRIPTION
**Pull request by tj-corrigan**
We still need to review this for content in addition to getting overall approval

### CODE REVIEW

**Comment by tj-corrigan** [comment]
@ryan-richt @ddgenome

**Comment by ddgenome** [comment]
TJ, it looks good to me. I committed a few minor changes. I'll make another comment in line.

File: _posts/2015-06-12-etcd-clustering.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+![etcd clustering architecture](/img/etcd-cluster-architecture.png)
+*[source](https://coreos.com/docs/cluster-management/setup/cluster-architectures/#production-cluster-with-central-services)*
+
+## The Bootstrapping Problem
+
+etcd requires an initial bootstrapping to form a cluster. This can be accomplished in [several ways](https://github.com/coreos/etcd/blob/master/Documentation/clustering.md). We ended up deciding to use the static method to reduce dependencies on external systems.
```
Comment (ddgenome) [inline _posts/2015-06-12-etcd-clustering.md]: Should we say something about the issues we had with https://discovery.etcd.io/ and how it does not address the member add/leave case?

**Comment by ddgenome** [comment]
I assigned the review to @slbahr

**Comment by slbahr** [comment]
I approve this blog.  One minor comment...When we reference our technologist should we have a link to their linked in or git hub profiles?  Kind of build followers for our technologists.  LMK what you think?

**Comment by tj-corrigan** [comment]
That’s a good idea, I’ll ask Phil what link he would like me to use.

From: Stephanie Bahr
Reply-To: MonsantoCo/engineering-blog
Date: Monday, June 8, 2015 at 10:13 AM
To: MonsantoCo/engineering-blog
Cc: "T.J. Corrigan"
Subject: Re: [engineering-blog] Etcd clustering (#19)


I approve this blog. One minor comment...When we reference our technologist should we have a link to their linked in or git hub profiles? Kind of build followers for our technologists. LMK what you think?

—
Reply to this email directly or view it on GitHub<https://github.com/MonsantoCo/engineering-blog/pull/19#issuecomment-110033366>.
This e-mail message may contain privileged and/or confidential information, and is intended to be received only by persons entitled
to receive such information. If you have received this e-mail in error, please notify the sender immediately. Please delete it and
all attachments from any servers, hard drives or any other media. Other use of this e-mail by you is strictly prohibited.

All e-mails and attachments sent and received are subject to monitoring, reading and archival by Monsanto, including its
subsidiaries. The recipient of this e-mail is solely responsible for checking for the presence of "Viruses" or other "Malware".
Monsanto, along with its subsidiaries, accepts no liability for any damage caused by any such code transmitted by or accompanying
this e-mail or any attachment.


The information contained in this email may be subject to the export control laws and regulations of the United States, potentially
including but not limited to the Export Administration Regulations (EAR) and sanctions regulations issued by the U.S. Department of
Treasury, Office of Foreign Asset Controls (OFAC).  As a recipient of this information you are obligated to comply with all
applicable U.S. export laws and regulations.

**Comment by ddgenome** [comment]
Should we remove the Pets vs. Cattle stuff since the approval for posting that video might be a ways off?

**Comment by tj-corrigan** [comment]
Sure if that is a ways off we can remove it. The other option is to replace it with a link to some other similar talk if anyone can think of one.

**Comment by ddgenome** [comment]
There is no timelines, so we should probably proceed with an alternate. The Pets vs. Cattle analogy is fairly widespread at this point.

http://www.slideshare.net/gmccance/cern-data-centre-evolution (origin?)
http://it20.info/2012/12/vcloud-openstack-pets-and-cattle/
https://blog.engineyard.com/2014/pets-vs-cattle
https://news.ycombinator.com/item?id=7311704

**Comment by jessitron** [comment]
Occassionaly

**Comment by jessitron** [comment]
I liked the part about Phil, and vote for linking his twitter. Or, has he given that talk publicly somewhere?

**Comment by jessitron** [comment]
"We ended up deciding to use" <- awkward. "We used"

**Comment by jessitron** [comment]
"instance metadata endpoint)." <- stray closing paren

**Comment by slbahr** [comment]
Please move ahead with posting this.  We will hold off on the Excite talks for now as I have feedback from legal on this.

**Pull request merged**